pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -3040,7 +3040,9 @@ let
         };
       };
 
-      mgt19937.typst-preview = callPackage ./mgt19937.typst-preview { };
+      mgt19937.typst-preview = throw ''
```

Comment:
Can we add `throw` in here?

Reply:
Apparently yes.

Reply:
Maybe it should be moved to the aliases section ?

Reply:
Sure ! Done.